frontend/src/actions/send/draw_response_action.ts

```diff
@@ -0,0 +1,20 @@
+import type { ConnectionRepository, Message } from '@/models/connection_repository/connection_repository'
+
+class DrawResponseMessage implements Message {
+    action: string
+    body: {
+        response: boolean
+    }
+
+    constructor() {
+        this.action = 'drawResponse'
+    }
+}
+
+function DrawResponseAction(repository: ConnectionRepository, response: boolean) {
+    const m = new DrawResponseMessage()
+    m.body.response = response
```

Comment:
Instead of creating the `DrawResponseMessage` and then setting its variable `response`, a better idea is to pass the response as parameter and let the constructor initializa the object directly with the needed value.

Reply:
done